recordedit/form.controller.js

```diff
@@ -104,6 +104,29 @@
             $log.info(response);
         }
 
+
+        /*
```

Comment:
I think so.  I don't see why we would normally want to store explicit empty strings in the DB.